win32ss/printing/base/winspool/printers.c

```diff
@@ -449,9 +449,17 @@ EnumPrintersA(DWORD Flags, PSTR Name, DWORD Level, PBYTE pPrinterEnum, DWORD cbB
  
     /* Ref: https://stackoverflow.com/questions/41147180/why-enumprintersa-and-enumprintersw-request-the-same-amount-of-memory */
     bReturnValue = EnumPrintersW(Flags, pwszName, Level, pPrinterEnum, cbBuf, pcbNeeded, pcReturned);
-    HeapFree(hProcessHeap, 0, pwszName);
+    TRACE("*pcReturned is '%d' and bReturnValue is '%d' and GetLastError is '%ld'.\n",
+          *pcReturned, bReturnValue, GetLastError());
 
-    TRACE("*pcReturned is '%d' and bReturnValue is '%d' and GetLastError is '%ld'.\n", *pcReturned, bReturnValue, GetLastError());
```

Comment:
Not necessary.

Reply:
> GetLastError()

Well, MSDN says HeapFree() sets LastError. Is that false?
(And the line is a bit long...)